tests/test_taxi.py

```diff
@@ -69,7 +67,7 @@ def test_car_is_register_in_admin(self):
 
 class GitignoreTests(TestCase):
     def test_gitignore_exist(self):
-        file_exists = os.path.exists('.gitignore')
+        file_exists = os.path.exists(".gitignore")
         assert file_exists
```

Comment:
In Django tests, it's recommended to use `self.assertTrue()` instead of the plain `assert` statement for better integration with the test runner and more informative error messages. Consider changing `assert file_exists` to `self.assertTrue(file_exists)`.